### ISSUE TITLE
USPP PW calculation aborts with no information

### ISSUE THREAD

**Issue by kirk0830** [deepmodeling/abacus-develop#4371]
### Describe the bug

see the following:
```
                                                                                     
                              ABACUS v3.6.5

               Atomic-orbital Based Ab-initio Computation at UStc                    

                     Website: http://abacus.ustc.edu.cn/                             
               Documentation: https://abacus.deepmodeling.com/                       
                  Repository: https://github.com/abacusmodeling/abacus-develop       
                              https://github.com/deepmodeling/abacus-develop         
                      Commit: 10c5281 (Thu Jun 13 09:35:14 2024 +0800)

 Thu Jun 13 15:01:35 2024
 MAKE THE DIR         : OUT.ABACUS/
 RUNNING WITH DEVICE  : CPU / Intel(R) Xeon(R) Platinum 8269CY CPU @ 2.50GHz
 UNIFORM GRID DIM        : 15 * 15 * 15
 UNIFORM GRID DIM(BIG)   : 15 * 15 * 15
 DONE(0.0462065  SEC) : SETUP UNITCELL
 DONE(0.138757   SEC) : SYMMETRY
 DONE(48.0371    SEC) : INIT K-POINTS
 ---------------------------------------------------------
 Ion relaxation calculations
 ---------------------------------------------------------
 SPIN    KPOINTS         PROCESSORS  
 1       5216            16          
 ---------------------------------------------------------
 Use plane wave basis
 ---------------------------------------------------------
 ELEMENT NATOM       XC          
 H       1           
 ---------------------------------------------------------
 Initial plane wave basis and FFT box
 ---------------------------------------------------------
 DONE(48.0426    SEC) : INIT PLANEWAVE
 MEMORY FOR PSI (MB)  : 9.63037
 DONE(48.0432    SEC) : LOCAL POTENTIAL
 DONE(48.0797    SEC) : NON-LOCAL POTENTIAL
 DONE(48.0885    SEC) : INIT BASIS
 -------------------------------------------
 STEP OF RELAXATION : 1
 -------------------------------------------
 START CHARGE      : atomic

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 0 PID 40 RUNNING AT dp-lbg-471-13210994
=   KILLED BY SIGNAL: 9 (Killed)
===================================================================================

```
please try with:
[3672fc4d-a8db-4875-b235-70a667cb85dc.zip](https://github.com/user-attachments/files/15815484/3672fc4d-a8db-4875-b235-70a667cb85dc.zip)

### Task list for Issue attackers (only for developers)

- [ ] Verify the issue is not a duplicate.
- [ ] Describe the bug.
- [ ] Steps to reproduce.
- [ ] Expected behavior.
- [ ] Error message.
- [ ] Environment details.
- [ ] Additional context.
- [ ] Assign a priority level (low, medium, high, urgent).
- [ ] Assign the issue to a team member.
- [ ] Label the issue with relevant tags.
- [ ] Identify possible related issues.
- [ ] Create a unit test or automated test to reproduce the bug (if applicable).
- [ ] Fix the bug.
- [ ] Test the fix.
- [ ] Update documentation (if necessary).
- [ ] Close the issue and inform the reporter (if applicable).

**Comment by YuLiu98** [comment]
Hi @kirk0830 , can you please supply all input files?

**Comment by kirk0830** [comment]
> Hi @kirk0830 , can you please supply all input files?

I am trying to do EOS test, so do you mean all input files I generated? for example including all volumes and all pseudopotentials? if so, here they are:
[H_USPP_EOS.zip](https://github.com/user-attachments/files/15839703/H_USPP_EOS.zip)

I will do more tests on ultrasoft pseudopotentials later, on more elements and pseudopotentials. But if you are interested in some specific elements, please tell me.

**Comment by YuLiu98** [comment]
> > Hi @kirk0830 , can you please supply all input files?
> 
> I am trying to do EOS test, so do you mean all input files I generated? for example including all volumes and all pseudopotentials? if so, here they are: [H_USPP_EOS.zip](https://github.com/user-attachments/files/15839703/H_USPP_EOS.zip)
> 
> I will do more tests on ultrasoft pseudopotentials later, on more elements and pseudopotentials. But if you are interested in some specific elements, please tell me.

I mean that the example you supply in the [3672fc4d-a8db-4875-b235-70a667cb85dc.zip](https://github.com/user-attachments/files/15815484/3672fc4d-a8db-4875-b235-70a667cb85dc.zip) is incomplete.

OK, I find it in the new folder, thanks.

**Comment by YuLiu98** [comment]
This issue can be solved by PR #4397 

This is the running result:
[issue4371.tar.gz](https://github.com/user-attachments/files/15841921/issue4371.tar.gz)
